core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/kubernetes/reconfigure-rabbitmq-app.yml

```diff
@@ -3,3 +3,70 @@
   include_tasks: utils/patch-statefulset.yml
   vars:
     image_regexp: 'rabbitmq:.*'
+- name: Change rabbitmq stateful set to use {{ image_registry_address }}
```

Comment:
I think this code is not needed at all, please take a look at `utils/patch-statefulset.yml`. We refactored it lately. What I can see here is some redundant code from before the refactor and it does not take into account init containers for example (which was and is a bug tbh).
Also `KUBECONFIG: /home/{{ admin_user.name }}/.kube/config` is no longer valid, we decided to remove to completely as it does not bring any benefits. We started using `/etc/kubernetes/admin.conf` exclusively and we want to continue that. :+1:

Reply:
which part do you mean is legacy method?